spec/openapi.yaml

```diff
@@ -1113,30 +1123,19 @@ components:
           type: string
           example: "Ba1kmCbifTa8X"
 
-    PythonPackageUpdateInfo:
+    PrusaLinkPackage:
       type: object
+      description: PrusaLink package version available to update
       properties:
         available_updates:
           type: array
```

Comment:
We should either make it an object, not an array, or bring back name and version back into list.